src/lib/aragon-core.js

```diff
@@ -6,9 +6,12 @@ const networks = require('./networks')
 
 IPFS_TIMEOUT = 600000 // 10min
 
-async function getLatestFromRepo (repo, network) {
+async function getLatestFromRepo (repo) {
+  const web3 = new Web3(provider())
```

Comment:
We're [already creating an instance of web3 afterwards in the same function](https://github.com/aragon/aragon-desktop/blob/658c7e5aab0e23aba09c42aea7ea9cba4095f82b/src/lib/aragon-core.js#L21), we should just use this one.

Reply:
True, wasn't sure if we wanted to use our node for some specific reason. Changed it now